lib/rmt/cli/base.rb

```diff
@@ -83,6 +85,17 @@ def subcommand_prefix
 
   private
 
+  def locked_method(&_block)
+    RMT::Lockfile.create_file
+    yield
+    RMT::Lockfile.remove_file
+  rescue RMT::Lockfile::ExecutionLockedError
+    puts 'Process is locked'
```

Comment:
This doesn't set the exit code and prints to `stdout` instead of `stderr`.
1. See #87;
2. Handle `RMT::Lockfile::ExecutionLockedError` exceptions in rescue block of `handle_exceptions` method like all other exceptions:
https://github.com/SUSE/rmt/blob/ae2ce34b23242ab56bc24e347d2e9022ae090d9b/lib/rmt/cli/base.rb#L38-L40
3. Raise `RMT::CLI::Error` to set the exit code.